### PR TITLE
FX2024 Domain & Capabilities update / Understand cloud usage and cost

### PR DESCRIPTION
**Pull request by arthurclares**
## 🛠️ Description
This PR updates all the capabilities that are part of the understand cloud usage and cost  domain according to the new FinOps framework.

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Yes (required for `dev` PRs)
> - [x] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### 📖 Did you update documentation?

> - [x] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### CODE REVIEW

File: docs/_docs/framework/capabilities/understand/reporting.md [hunk 1]
```diff
@@ -82,6 +87,8 @@ We also recommend learning about [how cost data is tracked, stored, and refreshe
 
 Understanding your cost data is critical to enable accurate and meaningful showback to all stakeholders.
 
+If there is the need to report from multiple sources, for example multiple cloud providers, configuration management database (CMDB), or  IT asset management (ITAM) systems, consider using the FinOps Open Cost & Usage Specification ([FOCUS project](https://finops.org/focus)) as the standard billing schema for this report. To learn why organizations needs it and why Microsoft belives in FOCUS, you can review the [FOCUS documentation](https://microsoft.github.io/finops-toolkit/focus) available on FinOps toolkit provided by Microsoft.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:90]:
```suggestion
If there is the need to report from multiple sources, for example multiple cloud providers, configuration management database (CMDB), or  IT asset management (ITAM) systems, consider using the FinOps Open Cost & Usage Specification ([FOCUS project](../../../focus/README.md)) as the standard billing schema for this report.
```

File: docs/_docs/framework/capabilities/understand/reporting.md [hunk 2]
```diff
@@ -113,7 +120,9 @@ As a starting point, we focus on tools available in the Azure portal and Microso
 - Use the different tools optimized to provide the details you need to understand the holistic picture of your resource cost and usage.
   - [Analyze resource usage metrics in Azure Monitor](https://learn.microsoft.com/azure/azure-monitor/essentials/tutorial-metrics).
   - [Review resource configuration changes in Azure Resource Graph](https://learn.microsoft.com/azure/governance/resource-graph/how-to/get-resource-changes).
-- If you need to build more advanced reports or merge cost data with other cloud or business data, [leverage the FinOps toolkit Power BI reports](../../../../_reporting/power-bi/README.md).
+- If you need to build more advanced reports or merge cost data with other cloud or business data, [leverage the FinOps toolkit Power BI reports](../../../../_reporting/power-bi/README.md) part of the open source FinOps toolkit provided by Microsoft..
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:123]:
```suggestion
- If you need to build more advanced reports or merge cost data with other cloud or business data, [leverage the FinOps toolkit Power BI reports](../../../../_reporting/power-bi/README.md) part of the open source FinOps toolkit provided by Microsoft.
```

File: docs/_docs/framework/capabilities/understand/reporting.md [hunk 3]
```diff
@@ -113,7 +120,9 @@ As a starting point, we focus on tools available in the Azure portal and Microso
...
+    - FinOps hubs will accelerate your development and help you focus on building the features you need rather than infrastructure.
+    - FinOps hubs includes a [Power BI report](https://microsoft.github.io/finops-toolkit/power-bi) that normalizes data to the FOCUS schema, which can be a good starting point.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:125]:
```suggestion
    - FinOps hubs includes a [Power BI report](../../../../reporting/power-bi) that normalizes data to the FOCUS schema, which can be a good starting point.
```

File: docs/_docs/framework/capabilities/understand/reporting.md [hunk 4]
```diff
@@ -142,12 +151,12 @@ At this point, you're likely productively utilizing the native reporting and ana
 
 ## 🍎 Learn more at the FinOps Foundation
 
-This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Data analysis and showback capability](https://www.finops.org/framework/capabilities/analysis-showback/) article in the FinOps Framework documentation.
+This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Reporting + analytics capability](https://www.finops.org/framework/capabilities/???/) article in the FinOps Framework documentation.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:154]:
```suggestion
This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Reporting and analytics capability](https://www.finops.org/framework/capabilities/reporting-analytics/) article in the FinOps Framework documentation.
```

File: docs/_docs/framework/capabilities/understand/reporting.md [hunk 5]
```diff
@@ -142,12 +151,12 @@ At this point, you're likely productively utilizing the native reporting and ana
...
 
 You can also find related videos on the FinOps Foundation YouTube channel:
 
 <!--[!VIDEO https://www.youtube.com/embed/{id}?list={list}]-->
-{% include video.html title="Data analysis and showback videos" id="CVTJLdcozj1eEpxT" list="PLUSCToibAswlDSQdehKhi7ysP2hmetigl" %}
+{% include video.html title="Reporting + analytics videos" id="CVTJLdcozj1eEpxT" list="PLUSCToibAswlDSQdehKhi7ysP2hmetigl" %}
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:159]:
```suggestion
{% include video.html title="Reporting and analytics videos" id="CVTJLdcozj1eEpxT" list="PLUSCToibAswlDSQdehKhi7ysP2hmetigl" %}
```

**Review by flanakin — CHANGES_REQUESTED** [review]

**Comment by arthurclares** [comment]
@flanakin All the changes are implemented. It's waiting on your approval now.

**Review by flanakin — APPROVED** [review]

**Pull request merged**